cloud

```diff
@@ -624,7 +630,15 @@ function cloud_functions_deploy() {
                           --entry-point kcidb_load_queue \
                           --env-vars-file "$env_yaml" \
                           --runtime python37 \
-                          --trigger-topic "${trigger_topic}" \
+                          --trigger-topic "${load_queue_trigger_topic}" \
+                          --memory 256MB \
+                          --timeout 540
+    cloud_function_deploy "$project" "${prefix}pick_notification" \
```

Comment:
Please use plural for the Cloud Function's name, to match the entry point's name and what it's doing. Same for withdrawing below.